src/constants_json_strings.py

```diff
@@ -194,18 +203,18 @@
 
 # Costs - Total per asset
 COST_TOTAL = "costs_total"
-COST_OM_TOTAL = "costs_om_total"
-COST_OM_FIX = "costs_cost_om"  # Fix asset operation/management costs per year, not depending on use
+COST_OPERATIONAL_TOTAL = "costs_om_total"
+COST_OM = "costs_cost_om"  # Fix asset operation/management costs per year, not depending on use
```

Comment:
costs cost om ?

Reply:
Yeah, we anyway have to go over the strings here again (and how they are parsed in the report). Will leave them as is for now.